server/serializers/subscriptions.go

```diff
@@ -178,14 +207,20 @@ type Reviewer struct {
 }
 
 type DeleteSubscriptionRequestPayload struct {
-	Organization string `json:"organization"`
-	Project      string `json:"project"`
-	EventType    string `json:"eventType"`
-	ServiceType  string `json:"serviceType"`
-	ChannelID    string `json:"channelID"`
-	MMUserID     string `json:"mmUserID"`
-	TargetBranch string `json:"targetBranch"`
-	Repository   string `json:"repository"`
+	Organization                 string `json:"organization"`
+	Project                      string `json:"project"`
+	EventType                    string `json:"eventType"`
+	ServiceType                  string `json:"serviceType"`
+	ChannelID                    string `json:"channelID"`
+	MMUserID                     string `json:"mmUserID"`
+	TargetBranch                 string `json:"targetBranch"`
+	Repository                   string `json:"repository"`
+	PullRequestCreatedBy         string `json:"pullrequestCreatedBy"`
+	PullRequestReviewersContains string `json:"pullrequestReviewersContains"`
```

Comment:
Same.

Reply:
these cannot be modified as these are the expected key names for AZD